### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by smileyhead**
Sorry about that; this one should be fine.

### CODE REVIEW

File: resources/hu/cemu.po [hunk 1]
```diff
@@ -411,23 +411,23 @@ msgstr "Érvénytelen „meta.xml” fájl."
 
 #: MainWindow.cpp:430
 msgid "Can't find the \"title_id\" in the given \"meta.xml\" file."
-msgstr "A megadott „meta.xml” fájlban nem található „title_id”."
+msgstr "A megadott „meta.xml” fájlban nem található „title_id.”"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Quotation marks at the end are now misplaced

File: resources/hu/cemu.po [hunk 2]
```diff
@@ -411,23 +411,23 @@ msgstr "Érvénytelen „meta.xml” fájl."
...
 
 #: MainWindow.cpp:448
 msgid "Can't find the \"title_version\" in the given \"meta.xml\" file."
-msgstr "A megadott „meta.xml” fájlban nem található „title_version”."
+msgstr "A megadott „meta.xml” fájlban nem található „title_version.”"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Quotation marks at the end are now misplaced

File: resources/hu/cemu.po [hunk 3]
```diff
@@ -411,23 +411,23 @@ msgstr "Érvénytelen „meta.xml” fájl."
...
 
 #: MainWindow.cpp:452
 msgid "The given \"meta.xml\" file is no update or DLC."
 msgstr "A megadott „meta.xml” fájl nem frissítés vagy DLC."
 
 #: MainWindow.cpp:457
 msgid "Can't find the \"longname_en\" in the given \"meta.xml\" file."
-msgstr "A megadott „meta.xml” fájlban nem található „longname_en”."
+msgstr "A megadott „meta.xml” fájlban nem található „longname_en.”"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Quotation marks at the end are now misplaced

**Comment by smileyhead** [comment]
Fixed the quotation marks.
As far as I know, the localisation should already be using the latest English template; I did do the "Catalogue > Update from POT file..." thing.

**Comment by smileyhead** [comment]
Ah yes, I see the problem now, thanks.
Could you please elaborate on what hermite is?

**Comment by smileyhead** [comment]
How about now?

File: resources/hu/cemu.po [hunk 4]
```diff
@@ -1,156 +1,295 @@
...
+#: GeneralSettings2.cpp:167
 msgid "Bilinear"
-msgstr "Bilineáris"
+msgstr "Simított"
 
-#: GeneralSettings2.cpp:165
+#: GeneralSettings2.cpp:167
 msgid "Bicubic"
-msgstr "Bikubikus"
+msgstr "Négyzetesen simított"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Does it actually translate to Bicubic?

File: resources/hu/cemu.po [hunk 5]
```diff
@@ -1,156 +1,295 @@
...
-#: GeneralSettings2.cpp:157
+#: GeneralSettings2.cpp:159
 msgid "Disable precompiled shaders"
 msgstr "Előre összeállított shaderek letiltása"
 
-#: GeneralSettings2.cpp:165
+#: GeneralSettings2.cpp:167
 msgid "Bilinear"
-msgstr "Bilineáris"
+msgstr "Simított"
```
Comment (Fs00) [inline resources/hu/cemu.po]: Does it actually translate to Bilinear?

**Comment by smileyhead** [comment]
They are not exact translations, but this is how they would be worded in Hungarian.

**Comment by Fs00** [comment]
But is it clear in Hungarian that 'Simított' refers to the bilinear algorithm? The user that reads that option must perfectly understand that we are referring to that specific scaling algorithm.

**Comment by smileyhead** [comment]
Yes.

**Comment by Fs00** [comment]
Ok. Thanks for the contribution!

**Pull request merged**